src/main.rs

```diff
@@ -189,6 +246,12 @@ fn main() -> ! {
         // CoAP poll
         let coap_socket = socket_set.get::<UdpSocket>(coap_socket_handle);
         match operation_mode {
+            OperationMode::TokenProvisioning(ref mut p) => {
+                p.poll(coap_socket);
+                if p.done() {
+                    operation_mode = operation_mode.provisioning();
```

Comment:
Local platform provisioning happens on a different device than Fobnail provisioning, so I think it should report success and enter indefinite idle state instead. This could change for remote platform provisioning but we're not implementing it yet.

Reply:
@krystian-hebel We can easily do that, however then we would have to get `rtthost` (or something else) working to re-connect RTT without reflashing firmware (`cargo-embed` doesn't allow that)

Reply:
Fair point.